src/formBuilder/utils.js

```diff
@@ -1064,23 +1070,28 @@ export function generateElementComponentsFromSchemas(parameters: {
         allFormInputs,
       );
 
+      const componentProps = {
+        name: elementPropArr[index].name,
+        required: elementPropArr[index].required,
+        hideKey,
+        path: `${path}_${elementPropArr[index].name}`,
+        definitionData,
+        definitionUi,
+        neighborNames: elementPropArr[index].neighborNames,
+        dependents: elementPropArr[index].dependents,
+        dependent: elementPropArr[index].dependent,
+        parent: elementPropArr[index].parent,
+      };
+      Object.keys(elementPropArr[index].uiOptions).forEach((key) => {
```

Comment:
Hmm, I'd thought there was a more concise way to do this, something halfway between what you've implemented here and Flow's now-disfavored spread approach. *shrug*

Reply:
Oh I could use Object.assign here